Sources/Core/Transaction/CodableTransaction.swift

```diff
@@ -188,6 +188,7 @@ public struct CodableTransaction {
         return self.envelope.encode(for: type)
     }
 
+    @available(*, deprecated, message: "Please use PolicyResolver instead")
```

Comment:
We might be able to remove this completely now (breaking versioning a little) instead of deprecating. My assumption is that this function fails in a lot of cases anyway - due to the force try’s.

Also for example my tests showed that some chains prefer to have the nonce included to make valid estimations/resolving. So that’s why I put resolving nonce first in the new implementation of the function.

Reply:
For exact that I'm agreed to broke backward capability and drop that function completely rather then marking it as deprecated. We'll have enough mess whilst 3.\*.\* lifetime and we'd better to reduce them as much as possible.